R/checkSummations.R

```diff
@@ -68,8 +72,14 @@ checkSummations <- function(mifFile, outputDirectory = ".", template = "AR6", su
       mutate(variable = names(checkVariables[i]))
     children <- filter(data, !!sym("variable") %in% checkVariables[[i]]) %>%
       rename("child" = !!sym("variable"), "childVal" = !!sym("value"))
-    comp <- left_join(parent, children, by = c("model", "scenario", "region", "unit", "period")) %>%
-      left_join(select(summationGroups, c("child", "factor")), by = c("child"), relationship = "many-to-many")
+    comp <- left_join(parent, children, by = c("model", "scenario", "region", "unit", "period"))
+
+    if (summationsFile == "extractVariableGroups") {
```

Comment:
to avoid that this fails with `summationsFile = NULL`:
```suggestion
    if (isTRUE(summationsFile == "extractVariableGroups")) {
```